### PR TITLE
Generate minutes - feedback secretarie

### PR DESCRIPTION
**Pull request by StuyckRuben**
on updating secretary, piece-part for minutes mentioning secretary will update secretary
https://kanselarij.atlassian.net/browse/KAS-4303

- [ ] merge https://github.com/kanselarij-vlaanderen/decision-report-generation-service/pull/4

### CODE REVIEW

File: app/controllers/agenda/agendaitems/agendaitem/index.js
Comment (sergiofenoll) [inline app/controllers/agenda/agendaitems/agendaitem/index.js:1]: **question**: why do we need these changes?

This is the controller for the agendaitem route, i.e. we're editing a specific agendaitem. The bulk of the changes occur in the `saveSecretary` action which deals with editing the secretary for said agendaitem/decision-activity.

I don't think we want to update the minutes of the meeting when editing a single agenditem's secretary. I am fairly certain that should only happen when the meeting's secretary changes.

File: app/components/meeting/edit-meeting-modal.js [hunk 1]
```diff
@@ -231,26 +231,44 @@ export default class MeetingEditMeetingComponent extends Component {
     this.args.meeting.numberRepresentation = this.numberRepresentation;
     this.args.meeting.mainMeeting = this.selectedMainMeeting;
 
-    if (this.enableDigitalAgenda) {
```
Comment (sergiofenoll) [inline app/components/meeting/edit-meeting-modal.js]: **question**: is moving this whole block outside of the feature flag intentional?

File: app/components/meeting/edit-meeting-modal.js [hunk 2]
```diff
@@ -231,26 +231,44 @@ export default class MeetingEditMeetingComponent extends Component {
...
-      const currentMeetingSecretary = yield this.args.meeting.secretary;
-      if (currentMeetingSecretary?.uri !== this.secretary.uri) {
-        this.args.meeting.secretary = this.secretary;
-        const decisionActivities = yield this.store.queryAll('decision-activity', {
-          'filter[treatment][agendaitems][agenda][created-for][:id:]':
-            this.args.meeting.id,
+    if (this.args.meeting.secretary != this.secretary) {
```
Comment (sergiofenoll) [inline app/components/meeting/edit-meeting-modal.js]: **chore**: compare URIs, not objects

File: app/components/meeting/edit-meeting-modal.js [hunk 3]
```diff
@@ -231,26 +231,44 @@ export default class MeetingEditMeetingComponent extends Component {
...
+        const newSecretaryPerson = yield this.secretary.person;
+        const newSecretaryPersonFullName = yield newSecretaryPerson.get('fullName');
+        latestMinutesPiecePart.value = latestMinutesPiecePart.value.replace(currentSecretaryPersonFullName, newSecretaryPersonFullName);
+        yield latestMinutesPiecePart.save();
+
+        const fileMeta = yield this.exportPdf.perform(minutes);
+        if (fileMeta) {
+          yield this.replaceMinutesFile(minutes, fileMeta.id);
+        }
+      }
```
Comment (sergiofenoll) [inline app/components/meeting/edit-meeting-modal.js]: **chore**: undo these changes

The feature to re-generate the minutes when changing the secretary was already added in https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1887, let's remove the changes here so we don't implement the same thing twice.

**Review by sergiofenoll — CHANGES_REQUESTED** [review]

File: app/controllers/agenda/minutes.js [hunk 4]
```diff
@@ -36,29 +37,81 @@ function renderAttendees(attendees) {
...
+      }
+      break;
+    case constants.DECISION_RESULT_CODE_URIS.INGETROKKEN:
+      text = intl.t("minutes-retracted", {
+        mededelingOrNota: capitalizeFirstLetter(mededelingOrNota)
+      })
+      break;
+    case constants.DECISION_RESULT_CODE_URIS.KENNISNAME:
+      text = intl.t("minutes-acknowledged", {
+        mededelingOrNota: capitalizeFirstLetter(mededelingOrNota)
```
Comment (sergiofenoll) [inline app/controllers/agenda/minutes.js]: **chore**: don't call `capitalizeFirstLetter` here

The string ends up looking like this: `De Vlaamse Regering neemt akte van Deze mededeling.`

**Review by ValenberghsSven — APPROVED** [review]
reviewed, changed code with Tom and tested

**Pull request merged**